build.patch

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Should we put an explanation somewhere as to the why we need to patch this?

I assume it's coming from Thrift generation? is there a relevant issue somewhere maybe?

Reply:
Good point I'll see if I can add some comments to the patch file.

Reply:
So instead of commenting on why we need the patch I committed 0e856d2 which switches out the the existing classmap generator so that we no longer need to do this patch.